CMSgov/design-system
# [WNMGDS-302] Fix MODULE_NOT_FOUND error

### Summary
- Fixes "MODULE_NOT_FOUND" error by switching to yarn workspaces and hoisting deps
- Upgrades chromedriver to the latest chrome stable version 83 to fix e2e tests locally

### How to test
- Make sure you have chrome updated to version 83
- Do a fresh install of npm packages (run `git clean -fdx` before yarn install)
- Run `yarn test:e2e`. If you get an error about chromedriver versions being 80, try doing a force download of the correct chromedriver version: https://www.npmjs.com/package/chromedriver
    - e2e tests are finicky, and its best not to move your cursor around when running them
